test/SafeProtocolMediator.spec.ts

```diff
@@ -0,0 +1,496 @@
+import hre from "hardhat";
+import { loadFixture } from "@nomicfoundation/hardhat-toolbox/network-helpers";
+import { expect } from "chai";
+import { SignerWithAddress } from "@nomicfoundation/hardhat-ethers/signers";
+import { TestModule } from "../typechain-types";
+import { ZeroAddress } from "ethers";
+
+describe("SafeProtocolMediator", async () => {
+    let deployer: SignerWithAddress, owner: SignerWithAddress, user1: SignerWithAddress, user2: SignerWithAddress;
+    const SENTINEL_MODULES = "0x0000000000000000000000000000000000000001";
+
+    async function deployContractFixture() {
+        [deployer, owner, user1, user2] = await hre.ethers.getSigners();
```

Comment:
those are not going to change with every function call so you can extract this outside of the function like it's done in the safe-contracts tests